controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -161,6 +162,19 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 			return reconcile.Result{}, err
 		}
 
+		if storageCluster.Spec.AllowRemoteStorageConsumers {
```

Comment:
Why do we need this check here. It is implicit that if we have a pre 4.19 consumer then we don't proceed